pywinauto/unittests/test_application.py

```diff
@@ -1393,6 +1401,11 @@ def test_get_list_of_windows_win32(self):
         window_titles = [win_obj.window_text() for win_obj in dlgs]
         self.assertTrue(self.window_title in window_titles)
 
+    def test_issue_760_desktop_win32(self):
+        focused_element = self.desktop.get_active()
+        if not isinstance(focused_element, HwndWrapper):
+            raise Exception("Received object is not an instance of HwndWrapper")
```

Comment:
Better way is using `self.assertEqual(type(focused_element), HwndWrapper)`. It is one line, and it will print which exact type in case of failure. Please do it for all such tests.